docs/api/middleware/keyauth.md

```diff
@@ -219,6 +219,7 @@ curl --header "Authorization: Bearer my-super-secret-key"  http://localhost:3000
 | SuccessHandler | `fiber.Handler`                          | SuccessHandler defines a function which is executed for a valid key.                                 | `nil`                         |
 | ErrorHandler   | `fiber.ErrorHandler`                     | ErrorHandler defines a function which is executed for an invalid key.                                | `401 Invalid or expired key`  |
 | KeyLookup      | `string`                                 | KeyLookup is a string in the form of "`<source>:<name>`" that is used to extract key from the request. | "header:Authorization"        |
+| AdditionalKeyLookups | `[]string` | If additional fallback sources of keys are required, they can be specified here in order of precedence | []string{} (empty) |
```

Comment:
I'd probably name this `ExtraKeyLookup`, `ExtraKeyLookups`,  or something like that?

Reply:
`FallbackKeyLookups` was the best I could come up with - indicates clearly that they are used _after_ `KeyLookup` in all cases.